pkgs/modules/java/default.nix

```diff
@@ -1,19 +1,31 @@
-{ pkgs, lib, ... }:
+{ pkgs-unstable, lib, ... }:
 
 let
-  graalvm = pkgs.graalvm17-ce;
+  pkgs = pkgs-unstable;
+  graalvm = pkgs.graalvm19-ce;
```

Comment:
So weird that  "19-ce" in "graalvm19-ce" is version "22.3", but that seems right:

![image](https://github.com/replit/nixmodules/assets/4804/e72a7b0d-fb28-47c1-a055-fb62fdd4c810)